### PR TITLE
- add kubectl context parameter support

### LINKED ISSUE #70: support context parameter
Currently we can not specify context by --context parameter to sniff. We can use it with native kubectl parameters like:

`kubectl get pods -n foo --context some-context`

I am volunteer to implement this feature.

### PR DESCRIPTION
**Pull request by mstrYoda**
Resolves #70

### CODE REVIEW

File: pkg/cmd/sniff.go [hunk 1]
```diff
@@ -166,7 +173,25 @@ func (o *Ksniff) Complete(cmd *cobra.Command, args []string) error {
...
+	} else {
+		currentContext, exists = o.rawConfig.Contexts[o.rawConfig.CurrentContext]
+	}
+
+	if !exists {
+		return errors.New("context doesn't exist")
+	}
+
+	o.restConfig, err = clientcmd.NewNonInteractiveDeferredLoadingClientConfig(
+		&clientcmd.ClientConfigLoadingRules{ExplicitPath: o.configFlags.ToRawKubeConfigLoader().ConfigAccess().GetDefaultFilename()},
```
Comment (eldadru) [inline pkg/cmd/sniff.go:190]: Can you please explain the purpose of this line? why we should specify the path of the kubectl config file? isn't the API perform this lookup on his own?

**Comment by eldadru** [comment]
@mstrYoda - thank you very much for taking the time and implementing this feature.

See my comment + did you test this change? both when passing a custom context and using the current active one?

**Comment by mstrYoda** [comment]
> @mstrYoda - thank you very much for taking the time and implementing this feature.
> 
> See my comment + did you test this change? both when passing a custom context and using the current active one?

Yes @eldadru I've tested it and it worked both way (backward compatible). I can upload a screen record for it as well.

File: pkg/cmd/sniff.go (hunk 1 above)
Comment (mstrYoda) [reply pkg/cmd/sniff.go:190]: Actually I thought the same thing but it didn't work without specifying that parameter.

**Pull request merged**